sdk/resourcemanager/Azure.ResourceManager.Core/README.md

```diff
@@ -1,59 +1,155 @@
 # Azure ResourceManager Core client library for .NET
 
-This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
+This package follows the [new Azure SDK guidelines]<---(https://azure.github.io/azure-sdk/general_introduction.html)--> which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
 
 ## Getting started 
 
 ### Install the package
 
-Install the Azure Resources management core library for .NET with [NuGet](https://www.nuget.org/):
+Install the Azure Resources management core library for .NET with [NuGet]<---(https://www.nuget.org/)-->:
 
 ```PowerShell
 Install-Package Azure.ResourceManager.Core -Version 1.0.0-beta.1
 ```
 
 ### Prerequisites
+Set up a way to authenticate to Azure with Azure Identity.
 
-* You must have an [Azure subscription](https://azure.microsoft.com/free/)
+Some options are:
+- Trough the [Azure CLI Login](https://docs.microsoft.com/cli/azure/authenticate-azure-cli).
+- Via [Visual Studio](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet#authenticating-via-visual-studio).
+- Setting [Environment Variables]<--(docs/AuthUsingEnvironmentVariables.md)-->.
+
+More information and different authentication approaches using Azure Identity can be found in [this document](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet).
 
 ### Authenticate the Client
 
-To create an authenticated client and start interacting with Azure resources, please see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md)
+The default option to create an authenticated client is to use `DefaultAzureCredential`. Since all management APIs go through the same endpoint, in order to interact with Resources only one top-level  `AzureResourceManagerClient` has to be created.
```

Comment:
```suggestion
The default option to create an authenticated client is to use `DefaultAzureCredential`. Since all management APIs go through the same endpoint, in order to interact with Resources only one top-level  `ArmClient` has to be created.
```